articles/quickstart/backend/php/01-authorization.md

```diff
@@ -1,251 +1,184 @@
-        }
-    }
+# Our Auth0 API's Identifier.
+AUTH0_AUDIENCE='YOUR_API_IDENTIFIER'
+```
 
-    // This endpoint doesn't need authentication
-    public function publicEndpoint() {
-        return array(
-            "status" => "ok",
-            "message" => "Hello from a public endpoint! You don't need to be authenticated to see this."
-        );
-    }
+As PHP isn't able to read our `.env` file by itself, we'll want to install a library to help with that. Although we'll be using a particular library for our sample application's purposes, any 'dotenv' loader of preference will work in a real-world application. From our project directory, let's run the following shell command to install the library:
 
-    public function privateEndpoint() {
-        return array(
-            "status" => "ok",
-            "message" => "Hello from a private endpoint! You need to be authenticated to see this."
-        );
-    }
-}
+```sh
+composer require vlucas/phpdotenv
 ```
 
-To include this class with Composer autoloading, add the following to your `composer.json`:
+Next, let's create the PHP source file we'll be using for these code samples, `index.php`, and let's configure an instance of the Auth0 PHP SDK for our sample application:
 
-```json
-{
-    // ...
-    "autoload": {
-        "psr-4": {
-            "App\\": "src/"
-        }
-    }
-}
+```php
+<?php
+
+// Import the Composer Autoloader to make the SDK classes accessible:
+require 'vendor/autoload.php';
+
+// Load our environment variables from the .env file:
+(Dotenv\Dotenv::createImmutable(__DIR__))->load();
+
+// Create a configuration object for the Auth0 PHP SDK:
+$auth0Configuration = new \Auth0\SDK\SdkConfiguration(
+    domain: $env['AUTH0_DOMAIN'],
+    clientId: $env['AUTH0_CLIENT_ID'],
+    clientSecret: $env['AUTH0_CLIENT_SECRET'],
+    audience: $env['AUTH0_AUDIENCE']
+);
+
+// Now instantiate the Auth0 class with the above configuration:
+$auth0 = new \Auth0\SDK\Auth0($auth0Configuration);
 ```
 
-## Protect API Endpoints
 
-<%= include('../_includes/_api_endpoints') %>
+### Authenticating the user
 
-The `before` hook from the **router** package can be used to configure which routes are to be protected. For example, you may wish to protect all routes under a URL of `/api/private`.
+For this sample application, we're focusing on [authorization](https://auth0.com/intro-to-iam/authentication-vs-authorization/). There's numerous routes you could go for authenticating your users before they hit your backend API for authorization, such as using [Auth0's SPA.js library](https://github.com/auth0/auth0-spa-js). This approach is demonstrated in [this Quickstart app accompanying Github project](https://github.com/auth0-samples/auth0-php-api-samples/). Regardless of the approach you take, this sample application expects you to pass your Access Token to it through a request parameter or header to work.
 
-```php
-// index.php
+### Authorizing a token
```

Comment:
Wouldn't this be "authorizing a request"?

Reply:
Good call 👍